tools/git/hooks/src/pre_commit.py

```diff
@@ -85,17 +83,12 @@ def is_checked_file_pattern(file_name):
 
 def is_ignored_file_pattern(file_name):
     ignored_patterns = [
-        "lib/FreeRTOS-Plus-POSIX/",
-        "lib/FreeRTOS-Plus-TCP/",
-        "lib/FreeRTOS/",
-        "lib/include/FreeRTOS_POSIX/",
-        "lib/include/private/threading_alt.h",
-        "lib/ota/portable",
-        "lib/pkcs11/",
-        "lib/secure_sockets/portable",
-        "lib/third_party/",
-        "lib/wifi/portable",
-        "tests/common/posix",
+        "freertos_kernel/",
+        "projects/",
+        "tools/",
+        "vendors/",
```

Comment:
This will ignore vendors/boards/<board_name>/<aws_project>/config_files, we don't really need these don't need to adhere, right? I can see this being a bit tricky now since we definately want to ignore vendors/<sdk>.

Reply:
Yes, those are not owned by us and therefore excluded from style check.